examples/platforms/cc1352/radio.c

```diff
@@ -1327,6 +1327,28 @@ otError otPlatRadioSetTransmitPower(otInstance *aInstance, int8_t aPower)
     return OT_ERROR_NONE;
 }
 
+/**
+ * Function documented in platform/radio.h
+ */
+otError otPlatRadioGetCCAEnergyDetectThreshold(otInstance *aInstance, int8_t *aThreshold)
```

Comment:
nit: we tend to prefer to camel case acronyms as well - how about: `otPlatRadioGetCCAEnergyDetectThreshold` -> `otPlatRadioGetCcaEnergyDetectThreshold`

Reply:
Done.